Ginger/GingerCoreNET/GingerCoreNET.csproj

```diff
@@ -247,8 +248,10 @@
     <PackageReference Include="Selenium.WebDriver.MicrosoftWebDriver" Version="10.0.17134" />
     <PackageReference Include="SharpAdbClient" Version="2.3.23" />
     <PackageReference Include="SSH.NET" Version="2020.0.1" />
+    <PackageReference Include="System.Data.DataSetExtensions" Version="4.5.0" />
```

Comment:
why those System.Data.DataSetExtensions and System.Data.OleDb were added?

Reply:
System.Data.OleDb removed
System.Data.DataSetExtensions for datatable